eng/targets/Npm.Common.targets

```diff
@@ -0,0 +1,75 @@
+<Project DefaultTargets="Build" InitialTargets="_CheckForInvalidConfiguration">
+
+  <PropertyGroup>
+    <PackageVersion Condition=" '$(VersionSuffix)' != '' ">$(VersionPrefix)-$(VersionSuffix)</PackageVersion>
```

Comment:
Identical to https://github.com/aspnet/AspNetCore/blob/master/eng/targets/Npm.Common.targets except the first 2 lines where the `PackageVersion` is calculated.

Reply:
This is going to use SemVer2 by default. Is that okay for NPM packages?

Btw, here is how Arcade computes this stuff https://github.com/dotnet/arcade/blob/9104c21492e1c6ed98105dd87388bca8f1f1abaa/src/Microsoft.DotNet.Arcade.Sdk/tools/Version.props and https://github.com/dotnet/arcade/blob/c0451509373be4842e44d97fd6b79bf51064e2bc/src/Microsoft.DotNet.Build.Tasks.Packaging/src/build/Packaging.targets

Reply:
Does look like semver v2 works with npm, although it does constraint the range specifier: https://github.com/npm/node-semver#prerelease-tags